tasks/update-ocp-tag/update-ocp-tag.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: update-ocp-tag
   labels:
-    app.kubernetes.io/version: "1.5.0"
+    app.kubernetes.io/version: "1.5.1"
```

Comment:
Hmm, nothing is changing in the task. So no version bump is necessary I'd say?

Reply:
nice catch, fixed